examples/basic_mysql_connection/index.js

```diff
@@ -13,10 +13,10 @@ var connection = mysql.createConnection({
 
 connection.connect();
 const sqlStatments=['SELECT COUNT(*) as "Total No of Todos" FROM todos',
-					'SELECT * FROM todos WHERE Done=1',
-					'SELECT * FROM todos WHERE Done=0',
-					'SELECT * FROM todos GROUP BY Due DESC',
-					'SELECT * FROM todos WHERE Id=(SELECT MAX(Id) FROM todos) ',
+					'SELECT * FROM todos WHERE Done',
+					'SELECT * FROM todos WHERE !Done',	
```

Comment:
Yeah, this is more like what I would do 🙂

I prefer the `NOT` keyword over the `!`, but both are equivalent (I just find the `NOT` keyword easier to read).